### PR TITLE
[MIG]12.0 mrp_production_request

### PR DESCRIPTION
**Pull request by ougc27**
- Change Procurement rule to Stock Rule

- Change the product.uom model to uom.uom

- Use message_subscribe instead message_subscribe_users because message_subscribe_users doesn't exist in V12.

- Add the procurement group to manufacture requests created from procurement run method.

### CODE REVIEW

**Comment by ougc27** [comment]
Travis is red because this module depends of stock_available_unreserved wich is still on PR

https://github.com/OCA/stock-logistics-warehouse/pull/526

File: mrp_production_request/__manifest__.py [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
                "step to Manufacturing Orders for better manufacture "
                "planification.",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
     "development_status": "Mature",
     "maintainers": ['lreficent'],
     "category": "Manufacturing",
     "website": "https://github.com/OCA/manufacture",
     "author": "Eficent,"
+              "JARSA Sistemas,"
```
Comment (LoisRForgeFlow) [inline mrp_production_request/__manifest__.py]: Thanks for the work!

A quick comment while this is WIP: a migration does not credit you as author, please remove this line.

Comment (ougc27) [reply mrp_production_request/__manifest__.py]: Done!

Thank you for your feedback!

**Comment by tbaden** [comment]
https://github.com/OCA/stock-logistics-warehouse/pull/526 is merged now.  you can rebase

**Comment by HviorForgeFlow** [comment]
ping @ougc27, waiting your rebase to review it.

**Comment by LoisRForgeFlow** [comment]
@ougc27 are you going to finish this work?

**Comment by bealdav** [comment]
Please @ougc27 could you give more information about your availability here ?
If not, we'll make another PR.

Thanks a lot

File: mrp_production_request/models/mrp_production.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        string="Manufacturing Request", copy=False, readonly=True)
+
+    def _generate_finished_moves(self):
+        """`move_dest_ids` is a One2many fields in mrp.production, thus we
+        cannot indicate the same destination move in several MOs (which most
+        probably would be the case with MRs).
+        Storing them on the MR and writing them on the finished moves as it
+        would happen if they were present in the MO, is the best workaround
+        without changing the standard data model."""
+        move = super(MrpProduction, self)._generate_finished_moves()
```
Comment (bealdav) [inline mrp_production_request/models/mrp_production.py:21]:
```suggestion
        move = super()._generate_finished_moves()
```

File: mrp_production_request/models/stock_warehouse_orderpoint.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class Orderpoint(models.Model):
+    _inherit = "stock.warehouse.orderpoint"
+
+    def _quantity_in_progress(self):
+        res = super(Orderpoint, self)._quantity_in_progress()
```
Comment (bealdav) [inline mrp_production_request/models/stock_warehouse_orderpoint.py:11]:
```suggestion
        res = super()._quantity_in_progress()
```

File: mrp_production_request/wizards/mrp_production_request_create_mo.py [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+        string="Products needed",
+        inverse_name="mrp_production_request_create_mo_id", readonly=True)
+    date_planned_start = fields.Datetime(
+        'Deadline Start', copy=False, default=fields.Datetime.now,
+        index=True, required=True,
+        states={'confirmed': [('readonly', False)]}, oldname="date_planned")
+
+    @api.model
+    def default_get(self, fields):
+        rec = super(MrpProductionRequestCreateMo, self).default_get(fields)
```
Comment (bealdav) [inline mrp_production_request/wizards/mrp_production_request_create_mo.py:73]:
```suggestion
        rec = super().default_get(fields)
```

**Comment by HviorForgeFlow** [comment]
@bealdav we pinged him twice and no answer back in last 2 month. For my part, you are free to superseed it. I need it merged too.

**Comment by LoisRForgeFlow** [comment]
Superseded here: https://github.com/OCA/manufacture/pull/404

**Pull request closed without merging**